uniswap_swap_bot/src/bot.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+} from "forta-agent";
+
+import { UNISWAP_ADDRESS, SWAP_EVENT } from "./constant";
+
+const handleTransaction: HandleTransaction = async (
+  txEvent: TransactionEvent
+) => {
+  const findings: Finding[] = [];
+
+  const tokenSwapEvent = txEvent.filterLog(SWAP_EVENT, UNISWAP_ADDRESS);
+  tokenSwapEvent.forEach((swapEvent) => {
+    const { sender, recipient, amount0, amount1 } = swapEvent.args;
+
+    console.log("deposit and output", amount0, amount1);
+    findings.push(
+      Finding.fromObject({
+        name: "Swap detected",
+        description: `${Number(amount0)} was swap ${amount1}`,
```

Comment:
The description could be more descriptive. Something along the lines of `"A 'swap' between ${token0} and ${token1} on UniswapV3 was detected"`, for example.